app/models/review.rb

```diff
@@ -2,6 +2,10 @@ class Review < ActiveRecord::Base
   belongs_to :user
   belongs_to :playspace
 
+  has_attached_file :photo, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```

Comment:
you should add a `missing.png` photo for each style: `original`, `medium`, `thumb`